components/Bounty/BountyAlert.tsx

```diff
@@ -23,7 +28,8 @@ const BountyAlert = ({ bounty, allBounties }: BountyAlertParams) => {
       return null;
     }
   } else if (allBounties.length) {
-    const firstBounty = new Bounty(allBounties[0]);
+    const firstBounty =
+      bountyType === "question" ? allBounties[0] : new Bounty(allBounties[0]);
```

Comment:
There is probably a use case but this feels weird because the same array can sometimes have bounties and other times raw data? Is it possible to covert `allBounties` to always be an array of bounty objects earlier on?